Demo/Sources/Players/PlaybackView.swift

```diff
@@ -435,12 +428,12 @@ private struct TimeSlider: View {
         ._debugBodyCounter(color: .blue)
     }
 
-    private static func formattedDuration(_ duration: TimeInterval) -> String {
-        if duration < 60 * 60 {
-            return shortFormatter.string(from: duration)!
+    private static func formattedDuration(_ time: TimeInterval, totalDuration: TimeInterval) -> String {
```

Comment:
Smart, this is intended to avoid frame changes when changing the progress I guess.

Since there is now a duration parameter we could maybe update the method prototype, something like `formattedTime(_ time: TimeInterval, duration: TimeInterval)`?